resources/views/hardware/view.blade.php

```diff
@@ -752,9 +752,10 @@ class="table table-striped snipe-table"
                   <th data-field="icon" data-visible="true" style="width: 40px;" class="hidden-xs" data-formatter="iconFormatter"></th>
                   <th class="col-sm-2" data-visible="true" data-field="created_at" data-formatter="dateDisplayFormatter">{{ trans('general.date') }}</th>
```

Comment:
I'm thinking we should probably make this hidden by default, since the created_at date is less relevant now? The action_date is the more important of them.

Reply:
Done. Thanks for your time reviewing this.